### ISSUE TITLE
Readme not in line with code

### ISSUE THREAD

**Issue by andnischneider** [emilhaegglund/SDBW#1]
Hi, 

we talked on the SBW conference last week, thanks again for answering my questions!

I just wanted to test your workflow and noticed that the readme and the code are not concordant. So when I change the config file as suggested in the readme, I get a bunch of KeyErrors and TypeErrors; for example when I change the "method" section from 

method: "prune_gtdb"

to 

method: 
    subsample_refseq: True
    subsample_gtdb: True
    prune_gtdb: True

as suggested in the readme, I get a TypeError from line 53 of download_genome_data.smk that dictionaries can't be concatenated. Is there any other way to turn on several of the three options or do I have to run separate runs?

Not sure if the readme is outdated or too far ahead here :) 

A second issue I had was that on my computer (Mac OS), I could not install diamond from the supplied yaml file but had to add 

pip:

before the diamond specification. Not sure if this is only a Mac OS issue or not..

Now that I have adjusted the config file and managed to install diamond, it seems to run just fine.

Cheers
Andreas

**Comment by emilhaegglund** [comment]
Hi,
Nice to meet you up in Umeå and thanks for putting up a great workshop, also thanks for trying out the workflow.
Oh, I must have forgotten to push the latest version of the README, I have updated it now, it was a bit outdated. You will have to run separate runs for the different sampling options in the current version.
Thanks for letting me know about the issue with Diamond on MacOS, I have only tested this on Linux so far. I have added a fix that should solve this.

Let me know if you run into more problems.

Cheers,
Emil

**Comment by andnischneider** [comment]
Thank you for the quick response and fix! 

I then got stuck at the prokka stage, which turned out to be entirely a Mac issue again, apparently the new hardware (M1 processor) causes some weird issues, and it took me quite a while to get the right setup for java, hmmer, and diamond to work. I had to set up a Rosetta 2 installation of homebrew (described here: https://stackoverflow.com/questions/64882584/how-to-run-the-homebrew-installer-under-rosetta-2-on-m1-macbook) and then install the programs through that on my system because conda refuses to install them no matter what..
I will test it on a Linux system as well to make sure it was only that causing problems. Regarding the Mac issues, it would probably be faster to containerize the workflow than to include workaround for all of that into the readme or the code.

The workflow finally ran through completely now and I will take a look at the output, and try the other methods as soon as I have time.

/Andreas
